### PR TITLE
Update mqtt-launcher.py

### PR DESCRIPTION
**Pull request by Aergernis**
Updated to new API in paho-mqtt 2.x

### CODE REVIEW

File: mqtt-launcher.py [hunk 1]
```diff
@@ -130,7 +133,7 @@ def on_disconnect(mosq, userdata, rc):
     transportType = cf.get('mqtt_transport_type', 'tcp')
 
     # initialise MQTT broker connection
-    mqttc = paho.Client(clientid, clean_session=False, transport=transportType)
+    mqttc = paho.Client(paho.CallbackAPIVersion.VERSION2, clientid, clean_session=True, transport=transportType)
```
Comment (jpmens) [inline mqtt-launcher.py:136]: Why did you change `clean_session` from False as default to True?

Comment (Aergernis) [reply mqtt-launcher.py:136]: Was testing with some options and forgot to change it back. False is fine.

Comment (jpmens) [reply mqtt-launcher.py:136]: If you change that back, I’ll gladly commit your PR.

Comment (Aergernis) [reply mqtt-launcher.py:136]: my be a dumb questin, but i'm not used to github.... can i edit it and the pull will update or do i have to mak a new request?

**Comment by jpmens** [comment]
>my be a dumb questin, but i'm not used to github.... can i edit it and the pull will update or do i have to mak a new request?

depends a bit how you originally submitted, but you ought to be able to edit, and it'll update the PR (until I merge it. :-)

**Comment by Aergernis** [comment]
i think it updated here in PR
